pyproject.toml

```diff
@@ -0,0 +1,75 @@
+[build-system]
```

Comment:
This replaces both setup.py and setup.cfg with "modern" python packaging practices.